### PR TITLE
Add support for highlighted rows

### PR DESCRIPTION
**Pull request by JaySunSyn**
See https://github.com/Neovici/cosmoz-omnitable/issues/10
See PR https://github.com/Neovici/cosmoz-omnitable/pull/49

### CODE REVIEW

**Comment by nomego** [comment]
I think we need an option to control whether we should highlight a single row or multiple rows. We have use cases for both as far as I know. Also please try to refactor to avoid code duplication, if possible.

**Comment by nomego** [comment]
Actually we might be able to / want to control single/multi behavior in omnitable instead.

**Comment by JaySunSyn** [comment]
Regarding codeclimate's issue: I think it doesn't make much sense to refactor it... Should I do it anyways?

I added the multi highlighting in omnitable https://github.com/Neovici/cosmoz-omnitable/pull/49

File: cosmoz-grouped-list.js [hunk 1]
```diff
@@ -456,6 +464,23 @@
 			}
 		},
 
+		highlightItem(item, reverse) {
+			var model = this._getModelFromItem(item),
+				highlightedIndex = this.highlightedItems.indexOf(item);
+
+			if (!this.isItemHighlighted(item) && !reverse) {
```
Comment (plequang) [inline cosmoz-grouped-list.js]: Maybe it's not necessary to call `isItemHighlighted` as this will call indexOf on highlighted items again.

**Comment by JaySunSyn** [comment]
Did that in `cosmoz-grouped-list-template` :/ but didn't copy the code from omnitables's bower_components. Sorry for that. @plequang

**Comment by plequang** [comment]
You should try bower link when you work in parallel on two components.
https://bower.io/docs/api/#link
http://cosmoz-frontend-rtd.readthedocs.io/en/latest/public/20-dependency-management.html#bower-link

**Comment by plequang** [comment]
I think we can keep row highlighting as it is in this PR for now, and we will see when using that new feature in real use cases if we miss something.

**Comment by JaySunSyn** [comment]
Thanks! Also added a demo now..

**Pull request merged**